<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test_SSH_Qt_2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="SshUpload">
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>SshUpload</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>IPv4 Address</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="ipadd1"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>History</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <column>
              <property name="text">
               <string>Check</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP Address</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="CheckAll">
             <property name="text">
              <string>CheckAll</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logsaveButton">
             <property name="text">
              <string>SaveLog</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="test2button">
          <property name="text">
           <string>SshTest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testButton">
          <property name="text">
           <string>PingTest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="excuteButton">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transferButton">
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFileButton">
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="selectFileLabel">
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AdbUpload">
       <attribute name="title">
        <string>AdbUpload</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>IPv4 Address</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="ipadd2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>ADB control</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="adbshellButton">
             <property name="text">
              <string>Shell</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adbpushButton">
             <property name="text">
              <string>Push</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adbkillButton">
             <property name="text">
              <string>Kill</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adbconnect_Button">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Settings">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>SSH UserName / SSH PassWord / SSH FilePath / SSH ExcutePath</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="username_ssh_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="password_ssh_1">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filepath_ssh_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="excutepath_ssh_1"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
